sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -2073,15 +2073,6 @@ class CSVSuite extends QueryTest with SharedSparkSession with TestCsvData {
     }
   }
 
-  test("do not produce empty files for empty partitions") {
```

Comment:
This has to be reverted for a different reason I pointed out in that PR https://github.com/apache/spark/pull/23052#issuecomment-440895059 as well.